sdks/python/apache_beam/utils/timestamp.py

```diff
@@ -29,6 +29,7 @@
 
 import dateutil.parser
 import pytz
+from google.protobuf import timestamp_pb2
```

Comment:
We should not have this import here. Google dependencies are installed optionally and may not be available. (Or is this coming through just protobuf dependency?)

Reply:
This is just through the protobuf dependency